pkg/env/vulnerability_management.go

```diff
@@ -13,4 +13,11 @@ var (
 	// ImageCVEEdgeCustomJoin is to enable/disable the adding custom join with image_cve_edges table.
 	// It is a temporary workaround to fix image_cve_edge search issue (ROX-17252)
 	ImageCVEEdgeCustomJoin = RegisterBooleanSetting("ROX_IMAGE_CVE_EDGE_CUSTOM_JOIN", true)
+
+	// OrphanedCVEsKeepAlive is to enable/disable preserving orphaned CVEs for a short interval before removing them
+	// from storage
+	OrphanedCVEsKeepAlive = RegisterBooleanSetting("ROX_ORPHANED_CVES_KEEP_ALIVE", false)
```

Comment:
Should we default to true?

Reply:
I think we keep it disabled and only have the affected customer turn it on, at least for the patch.